recipe/meta.yaml

```diff
@@ -48,7 +49,7 @@ requirements:
     - pandas >=1.2
     - packaging
```

Comment:
Is this really needed? Should it go in `host` instead?

Reply:
Yes, it's needed, it's used for version comparisons a bunch.

Reply:
Ok, then you might want to update the dependencies in panel directly :)